cmd/list/environment.go

```diff
@@ -0,0 +1,102 @@
+package list
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/dream11/odin/pkg/constant"
+	"strconv"
+
+	"github.com/dream11/odin/internal/service"
+	"github.com/dream11/odin/pkg/table"
+	environment "github.com/dream11/odin/proto/gen/go/dream11/od/environment/v1"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var name string
```

Comment:
Enable golint for the repo. I guess it might complain about var names being small

Reply:
It is already enabled
https://github.com/dream11/odin/blob/feat/cobra/.github/workflows/go-lint.yaml